<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57); color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size:11pt;</string>
      </property>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Expected Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Workflow</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deprecated</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
      </property>
      <property name="text">
       <string>Tasks for the day</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QWidget" name="buttons_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(64, 102, 72);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="note">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;
</string>
         </property>
         <property name="text">
          <string>Note</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_book">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>Book Time / Task Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="usertype_label">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(4, 255, 0);color:rgb(117, 2, 2);font-size:12pt;</string>
      </property>
      <property name="text">
       <string>User</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(64, 102, 72);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_prior_week">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>Week before</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_next_week">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>Next Week</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
